src/Rule/LatteTemplatesRule.php

```diff
@@ -131,7 +132,24 @@ private function analyseTemplates(array $templates, Scope $scope, array &$errors
                 /** @phpstan-var CollectedIncludePathArray $data */
                 $data = $collectedData->getData();
                 $collectedIncludedPath = CollectedIncludePath::fromArray($data, $this->typeStringResolver);
-                $includeTemplates[] = new Template($dir . '/' . $collectedIncludedPath->getPath(), $template->getActualClass(), array_merge($collectedIncludedPath->getVariables(), $template->getVariables()), $template->getComponents());
+                if (strpos($templatePath, '*') !== false) {
+                    /** @var string $file */
+                    foreach (Finder::findFiles($templatePath)->from($dir) as $file) {
+                        $includeTemplates[] = new Template(
```

Comment:
There are not included files or are they? Maybe this should be few rows above in main foreach

Reply:
It shoukd bw removed I was testing it for inckuded files too but was not able to make it useful. This is forgotten relict